google-cloud-bigquery/acceptance/bigquery/table_test.rb

```diff
@@ -476,6 +498,37 @@
     data.next.must_be :nil?
   end
 
+  it "inserts rows asynchonously with insert_ids option" do
+    insert_result = nil
+
+    inserter = table.insert_async insert_ids: insert_ids do |result|
+      insert_result = result
+    end
+    inserter.insert rows
```

Comment:
Why are `insert_ids` passed to the with the rows? The inserter object could be used to insert thousands of rows, users probably won't know all of the `insert_ids` when the inserter object is created.

```ruby
inserter = table.insert_async do |result|
  insert_result = result
end
inserter.insert rows, insert_ids: insert_ids
```

I was expecting `AsyncInserter#insert` to be the same signature as `Dataset#insert` and `Table#insert` and `Table#async_insert`

Reply:
Oh my, of course, great catch. I'll switch to more realistic testing as well when I fix this.